static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -267,6 +324,30 @@ define([
                     } else {
                         resolve();
                     }
+
+                    if (abVideoAutoplay) {
+                        // Annoyingly we pass the `parentNode` as the video is absolutely positioned.
+                        var parentNode = player.el().parentNode;
+                        var firstAutoplay = true;
+                        var elementInView = ElementViewable(parentNode, parentNode.clientHeight * (3 / 4), function () {
+                            if (firstAutoplay) {
+                                player.volume(0);
+                                firstAutoplay = false;
+                            }
+                            player.play();
+                        });
+                        elementInView.on('viewenter', function autoplayInView() {
+                            if (firstAutoplay) {
```

Comment:
I am trying to avoid moving any of this to instantiation to keep it separate as it's an A/B test.
We can look at a better implementation once we have results.